src/components/ViewDao/UnknownSchemeCard.scss

```diff
@@ -0,0 +1,88 @@
+.wrapper {
+	background: white;
```

Comment:
weird indentation here, tabs?

Reply:
It seems that the tabs predate my changes:  They came from SchemeCard.scss.

Reply:
So I went ahead and removed the tabs from this file, but will not do the same for other files.